### ISSUE TITLE
map query options

### ISSUE THREAD

**Issue by zwhitman** [DataUSA/datausa-site#193]
as an enhancement i think it'd be useful to allow for data subsets. so map and zoom to these specific states, or these specific counties, etc)

**Comment by alexandersimoes** [comment]
Any idea what the interaction design around this would be? I also think it would be great to have a zoomed in view with detailed data of just one state but not sure how we can keep that "simple" and launchable within the time frame. Where you thinking it would happen onclick of a states geography?

**Comment by zwhitman** [comment]
yeah - totally depends on how far we want to take it and how much time is available. one method could be on-state select. this could zoom to state and then auto-display that state's lower geography data. This would prob be the simplest implementation. all that said, i think this could also be controlled by some sort of panel (maybe along the top or sidebar right). Whatever we decide, i think it's important to keep map interactions consistent - so when you select something you know what'll happen and won't lose what you're doing. So if you want to zoom into new england, you don't lose everything but new hampshire
